diff --git a/src/HFM.Preferences/Properties/Settings.Designer.cs b/src/HFM.Preferences/Properties/Settings.Designer.cs
index 0c2b634..01faf83 100644
--- a/src/HFM.Preferences/Properties/Settings.Designer.cs
+++ b/src/HFM.Preferences/Properties/Settings.Designer.cs
@@ -346,7 +346,7 @@ namespace HFM.Preferences.Properties {
         
         [global::System.Configuration.UserScopedSettingAttribute()]
         [global::System.Diagnostics.DebuggerNonUserCodeAttribute()]
-        [global::System.Configuration.DefaultSettingValueAttribute("notepad.exe")]
+        [global::System.Configuration.DefaultSettingValueAttribute("xdg-open")]
         public string LogFileViewer {
             get {
                 return ((string)(this["LogFileViewer"]));
@@ -358,7 +358,7 @@ namespace HFM.Preferences.Properties {
         
         [global::System.Configuration.UserScopedSettingAttribute()]
         [global::System.Diagnostics.DebuggerNonUserCodeAttribute()]
-        [global::System.Configuration.DefaultSettingValueAttribute("explorer.exe")]
+        [global::System.Configuration.DefaultSettingValueAttribute("xdg-open")]
         public string FileExplorer {
             get {
                 return ((string)(this["FileExplorer"]));
diff --git a/src/HFM.Preferences/Properties/Settings.settings b/src/HFM.Preferences/Properties/Settings.settings
index fc52f4f..2b58339 100644
--- a/src/HFM.Preferences/Properties/Settings.settings
+++ b/src/HFM.Preferences/Properties/Settings.settings
@@ -84,10 +84,10 @@
       <Value Profile="(Default)">Standard</Value>
     </Setting>
     <Setting Name="LogFileViewer" Type="System.String" Scope="User">
-      <Value Profile="(Default)">notepad.exe</Value>
+      <Value Profile="(Default)">xdg-open</Value>
     </Setting>
     <Setting Name="FileExplorer" Type="System.String" Scope="User">
-      <Value Profile="(Default)">explorer.exe</Value>
+      <Value Profile="(Default)">xdg-open</Value>
     </Setting>
     <Setting Name="ProjectDownloadUrl" Type="System.String" Scope="User">
       <Value Profile="(Default)">http://fah-web.stanford.edu/psummary.html</Value>
diff --git a/src/HFM.Preferences/app.config b/src/HFM.Preferences/app.config
index b66efd8..9232c7d 100644
--- a/src/HFM.Preferences/app.config
+++ b/src/HFM.Preferences/app.config
@@ -80,10 +80,10 @@
                 <value>Standard</value>
             </setting>
             <setting name="LogFileViewer" serializeAs="String">
-                <value>notepad.exe</value>
+                <value>xdg-open</value>
             </setting>
             <setting name="FileExplorer" serializeAs="String">
-                <value>explorer.exe</value>
+                <value>xdg-open</value>
             </setting>
             <setting name="ProjectDownloadUrl" serializeAs="String">
                 <value>http://fah-web.stanford.edu/psummary.html</value>
diff --git a/src/HFM/app.config b/src/HFM/app.config
index dd3c580..44ce2a0 100644
--- a/src/HFM/app.config
+++ b/src/HFM/app.config
@@ -93,10 +93,10 @@
             <value>Standard</value>
          </setting>
          <setting name="LogFileViewer" serializeAs="String">
-            <value>notepad.exe</value>
+            <value>xdg-open</value>
          </setting>
          <setting name="FileExplorer" serializeAs="String">
-            <value>explorer.exe</value>
+            <value>xdg-open</value>
          </setting>
          <setting name="ProjectDownloadUrl" serializeAs="String">
             <value>http://fah-web.stanford.edu/psummary.html</value>
